frontend/src/metabase/dashboard/components/DashboardHeader/DashboardHeader.tsx

```diff
@@ -641,7 +641,7 @@ class DashboardHeaderContainer extends Component<DashboardHeaderProps> {
 
   saveAsPDF = async () => {
     const { dashboard } = this.props;
-    const cardNodeSelector = "#Dashboard-Cards-Container";
+    const cardNodeSelector = "#Dashboard-Parameters-And-Cards-Container";
```

Comment:
Nit: extracting this to a single constant that we share between this component and `Dashboard.tsx` would be nice